client/messaging/message-input.tsx

```diff
@@ -148,19 +277,75 @@ export const MessageInput = React.forwardRef<MessageInputHandle, MessageInputPro
       }),
     })
 
+    const onUserMentionSelect = useStableCallback((user: SbUser) => {
```

Comment:
Stop writing new `useCallback`/`useStableCallback`. The only time to `useCallback` now is if you get the lint warning about it making a `useMemo` useless (and hopefully React fixes that warning soon)

Reply:
Also inline this